doc/api/addons.md

```diff
@@ -11,8 +11,7 @@ provide an interface between JavaScript running in Node.js and C/C++ libraries.
 There are three options for implementing Addons: N-API, nan, or direct
 use of internal V8, libuv and Node.js libraries. Unless you need direct
 access to functionality which is not exposed by N-API, use N-API.
-Refer to the section [C/C++ Addons - N-API](n-api.html)
-for more information on N-API.
+Refer to the [N-API documentation](n-api.html) for more information on N-API.
```

Comment:
`C/C++ Addons - N-API` is how the N-API documentation appears in the [TOC](https://github.com/nodejs/node/blob/55e0ad9ae690f6d73b0caff7b813eb4d61195b94/doc/api/index.md#L17).

Reply:
Yeah, the hyphen there is ambiguous to me. Is it supposed to say that it's a subsection of the C/C++ Addons doc. But really, it's the N-API doc, and I'm guessing the only reason it's not being called N-API is because they want people to find it if they're looking for C/C++ Addons or if the see the other C/C++ Addons doc.

Reply:
Anyway, I can revert that change if it makes you or anyone else uncomfortable.

Reply:
I not opposed to rewording but I'd like it to be consistent with the TOC.

Reply:
Ooooohhhhhhhh.... I see. Updated.